.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,25 @@
-
 ### What does this PR do?
 
-<!-- A brief description of the change being made with this pull request. -->
+- [ ] ✋ I am aware of the [CONTRIBUTING](https://github.com/aws-ia/terraform-aws-eks-blueprints/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests
```

Comment:
Lets see if this helps reinforce the practice to flesh out details through issues first before committing to a large body of code in a PR